src/Hooks.cpp

```
@@ -165,44 +165,72 @@ namespace Hooks
 
-		//
-		BoneLimitFix code(GeometrySkinningBoneFix.address() + 7);
-
-		//
 		auto& localTrampoline = SKSE::GetTrampoline();
-		SKSE::AllocTrampoline(14 + code.getSize());
+		SKSE::AllocTrampoline(14 + sizeof(patch));
 
-		//
-		localTrampoline.write_branch<5>(GeometrySkinningBoneFix.address(), localTrampoline.allocate(code));
+		void* code = localTrampoline.allocate(sizeof(patch));
```

Clang (14.0.6)

[warning] variable 'code' is not initialized

(cppcoreguidelines-init-variables)